pyproject.toml

```diff
@@ -23,3 +24,7 @@ build-backend = "setuptools.build_meta:__legacy__"
 # Uncomment if pyproject.toml worked fine to ensure consistency with flake8
 # line-length = 120
 target-version = ["py37", "py38", "py39", "py310"]
+
+
+[tool.usort]
+first_party_detection = false
```

Comment:
curious: what motivated this config?

Reply:
I decided to add it, as both [torchrec](https://github.com/pytorch/torchrec/blob/main/pyproject.toml#L3) and [torchvision](https://github.com/pytorch/vision/blob/main/pyproject.toml#L3) use it, so I though `pytorch` should follow their style for consistency.

[Here is some information](https://github.com/facebookexperimental/usort/issues/30) on what this setting does.